app/2.0/start/first-element/intro.md

```diff
@@ -95,30 +96,15 @@ components.
     demo/
     icon-toggle-finished/
     icon-toggle.html
+    index.html
     </pre>
 
 2.  Run the Polymer development server from the project directory:
 
-        polymer serve
-
-3.  `polymer serve` gives you two URLs to choose from - one for an application, another for 
-reusable components. Open the URL for **reusable components**. 
-
-    <img src="/images/2.0/first-element/polymer-serve-urls.png" alt="Output from the polymer serve 
-command, showing URLs" title="Local server URLs">
-
-    (Note that the path uses `icon-toggle`—the
```

Comment:
I wonder if this note should be retained in some form. Although they don't need to type in this URL, it'll show up in the URL bar, and this URL confuses some people.

Maybe add as a note `{.alert .alert-info}` at the very end?